app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/ElementOnActivityResultUseCase.java

```diff
@@ -0,0 +1,31 @@
+package org.eyeseetea.malariacare.domain.usecase;
```

Comment:
It sounds a bit strange to me this Use Case and the following one. Actually, they just execute in a separate thread one thing. Shouldn't this be part of the Controller? This is like managing the result of the post-send action, so I would expect the controller to have some error treatment embedded. Can you think again how t refactor these use cases?